case_studies/dc2_cataloging/run_great_lake/run_bliss_machine.sh

```diff
@@ -0,0 +1,4 @@
+sbatch --job-name=great_lake_exp_06-16-1 \
+       --output=great_lake_exp_06-16-1.out \
+       --error=great_lake_exp_06-16-1.err \
+       run_bliss_machine.sbatch exp_06-16-1
```

Comment:
It's great you figured this out! It should be easier to perform replicate runs now, which will give us more confidence in our scores.